bin/.travis/test.sh

```diff
@@ -68,50 +74,29 @@ if [ "$REUSE_VOLUME" = "0" ]; then
     fi
 fi
 
-printf "\nMake sure Node.js and Yarn are included in latest-node and latest-dev\n"
+printf "\nMake sure Node.js and Yarn are included in latest-node\n"
 docker -l error run -a stderr ez_php:latest-node node -e "process.versions.node"
-docker -l error run -a stderr ez_php:latest-dev node -e "process.versions.node"
 docker -l error run -a stderr ez_php:latest-node bash -c "yarn -v"
-docker -l error run -a stderr ez_php:latest-dev bash -c "yarn -v"
-
-printf "\nMinimal testing on ez_php:latest for use with ez user\n"
-docker run -ti --rm \
-  -v $(pwd)/volumes/ezplatform:/var/www \
-  -v $(pwd)/bin/.travis/testSymfonyRequirements.php:/var/www/testSymfonyRequirements.php \
-  ez_php:latest \
-  bash -c "php testSymfonyRequirements.php"
-
-
-printf "\nMinimal testing on ez_php:latest-dev for use with ez user\n"
-docker run -ti --rm \
-  -v $(pwd)/volumes/ezplatform:/var/www \
-  -v $(pwd)/bin/.travis/testSymfonyRequirements.php:/var/www/testSymfonyRequirements.php \
-  ez_php:latest-dev \
-  bash -c "php testSymfonyRequirements.php"
```

Comment:
Looking at this again, was there no way to adapt the script to make it work on both v2 and v3?

So due to this below usefulness is so so, but if ICU version is right now I guess we could have enabled this part of the test again.
> // Skip temporarily while ICU is an issue: https://github.com/sensiolabs/SensioDistributionBundle/issues/277
//$minorProblems = $symfonyRequirements->getFailedRecommendations();

Reply:
The current test was based on SensioDistributionBundle, not available in v3, it won't work without additional packages added to the meta. 

We could:
- Add https://github.com/symfony/requirements-checker to the meta - it works with v2 and v3 and comes with a `bin/requirements-checker` executable
- Wait until Composer 2 is released and use Platform check: https://php.watch/articles/composer-platform-check

We can try with Option 1, can it be done as a follow-up to this PR?

Reply:
> Wait until Composer 2 is released and use Platform check: https://php.watch/articles/composer-platform-check

This is nice for a few issues over the years (different php on build vs prod), but it will just do the same test as is done in install / update / create-project based on what is defined in composer.json. So we should be covered by this already.

Lets skip it for now, and as you propose evaluate 1 as a followup 👍